templates/patterns/dropdown/dropdown.ui_patterns.yml

```diff
@@ -1,71 +1,70 @@
 dropdown:
-  label: "Dropdown"
-  description: "Toggle contextual overlays for displaying lists of links and more with the Bootstrap dropdown plugin. https://v5.getbootstrap.com/docs/5.0/components/dropdowns/"
+  label: 'Dropdown'
+  description: 'Toggle contextual overlays for displaying lists of links and more with the Bootstrap dropdown plugin. For more information visit: https://v5.getbootstrap.com/docs/5.0/components/dropdowns/'
   settings:
     link:
       type: boolean
-      label: link
-      description: Show as a link
+      label: 'link'
+      description: 'When enabled, shows dropdown as a link. False by default.'
       preview: false
     direction:
       type: select
-      label: Direction
+      label: 'Direction'
+      description: 'Sets the direction of the dropdown (if not informed, dropdown by default):'
       options:
-        dropdown: "dropdown"
-        dropup: "dropup"
-        dropend: "dropend"
-        dropstart: "dropstart"
+        dropup: 'dropup'
```

Comment:
Missing dropdown option??

Reply:
I removed because I checked how the pattern defines the field. But adding it back, let's see if it makes more sense (and added the default comment in the option)